dashboard/app/views/projects/index.html.haml

```diff
@@ -11,6 +11,7 @@
   end
 
   projects_data[:specialAnnouncement] = Announcements.get_announcement_for_page("/projects")
+  projects_data[:showPublicGalleryUpdatedInfo] = current_user
```

Comment:
Is this supposed to be `!!current_user`? Or, maybe simpler, just grab `pageConstants.isSignedIn` from redux instead of having to pass this thru? I guess this is probably temporary / will be reverted, so whatever's easier.

Reply:
Thanks! I don't think `pageConstants` are set when at '/projects' since they're set during level init.
https://github.com/code-dot-org/code-dot-org/blob/0e810f695ed3d979224b2e242865340c65d04bf6/apps/src/redux/pageConstants.js#L119

I updated to `!!current_user` here. Thanks!

Reply:
Actually, I think I could use `isSignedIn` from `currentUserRedux`. `PersonalProjectsTable` uses this value.
Since this is temporary, I think I'll leave as is, but just wanted to leave a note here. Thanks!

Reply:
Good to know, thanks for noting! What's available where in Redux is always hard for me to remember :)